src/Microsoft.DotNet.Interactive.Formatting.Tests/HtmlFormatterTests.cs

```diff
@@ -192,28 +184,42 @@ public void It_displays_exceptions_thrown_by_properties_in_the_property_value_ce
 
                 writer.ToString()
                       .Should()
-                      .Contain("<td>{ System.Exception:");
+                      .Contain("<td>System.Exception: not ok");
             }
 
             [Fact]
-            public void Properies_of_System_Type_instances_are_not_expanded()
+            public void Properies_of_System_Type_instances_are_not_expanded_and_link_is_not_added()
             {
-                var formatter = HtmlFormatter.GetBestFormatterFor(typeof(Type));
+                var formatter = HtmlFormatter.GetPreferredFormatterFor(typeof(Type));
+
+                var writer = new StringWriter();
+
+                formatter.Format(typeof(Dummy.DummyNotInSystemNamespace), writer);
+
+                writer.ToString()
+                      .Should()
+                      .Be("Dummy.DummyNotInSystemNamespace");
+            }
+
+            [Fact]
+            public void Properies_of_System_Type_instances_are_not_expanded_and_link_is_added()
```

Comment:
I'd suggest including a "when ... " in the names of this test and the previous one in order to embed more specification into the tests. Why are there these two different outcomes?

Reply:
Thanks, made this clearer now